src/service-object.ts

```diff
@@ -34,6 +34,8 @@ import {
 export type RequestResponse = [Metadata, r.Response];
 
 export interface ServiceObjectParent {
+  interceptors: Interceptor[];
+  getRequestInterceptors(): Function[];
```

Comment:
The function is executed with the options from the request library we use, and expects to receive the same type back. Trying to write [this function](https://github.com/googleapis/nodejs-common/blob/0467b31cd37dd6829af1e32e4344f5c86d93010d/src/service.ts#L143) had the linter very angry, so the plain "Function[]" made it in with a previous PR. If anyone is able to help get me started by rewriting that function signature in a way that pleases the linter, it would be nice to have it done correctly.